app/v3/controllers/validators/SubmitEndOfPeriodStatementValidatorFactory.scala

```diff
@@ -57,8 +60,14 @@ class SubmitEndOfPeriodStatementValidatorFactory {
 
       private def validateMore(parsed: SubmitEndOfPeriodStatementRequestData): Validated[Seq[MtdError], SubmitEndOfPeriodStatementRequestData] = {
         import parsed.body._
+
+        val resolveStartDate = new ResolveStartDate(minYear)
```

Comment:
these can always go up above to replace lines 34 & 35, then just pass in 1900 and 2100 instead of the vals